deploy/docker/django/Dockerfile

```
@@ -1,28 +1,47 @@
+COPY ../../qatrack ./qatrack
+# Install dependencies
+RUN pip install ".[docker,postgres]" django-extensions
 
-FROM python:3.6
+# Runtime stage
+FROM python:3.12-slim-bookworm
 
-RUN echo 'deb http://apt-archive.postgresql.org/pub/repos/apt/ stretch-pgdg main' > /etc/apt/sources.list.d/pgdg.list
-# or change to 'deb http://apt.postgresql.org/pub/repos/apt/ $(grep -oP 'VERSION_CODENAME=\K\w+' /etc/os-release)-pgdg main' to support non stretch versions
-RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
 RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)